synapse/handlers/federation.py

```diff
@@ -946,6 +946,12 @@ async def on_invite_request(
         )
 
         context = EventContext.for_outlier(self._storage_controllers)
+
+        if event.is_notifiable:
```

Comment:
I don't think we need this conditional, as it'll always be true (since its an invite)?